### PR TITLE
Fixed the az cli issue and release version 2 with updated gem.

### PR DESCRIPTION
**Pull request by ashiqueps**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: management/azure_mgmt_network/azure_mgmt_network2.gemspec [hunk 1]
```diff
@@ -9,20 +9,20 @@ require '../azure_mgmt_network/lib/module_definition'
...
 Gem::Specification.new do |spec|
-  spec.name          = 'azure_mgmt_network'
+  spec.name          = 'azure_mgmt_network2'
   spec.version       = Azure::Network::Mgmt::VERSION
   spec.authors       = 'Microsoft Corporation'
   spec.email         = 'azrubyteam@microsoft.com'
   spec.description   = 'Microsoft Azure Network Management Client Library for Ruby'
   spec.summary       = 'Official Ruby client library to consume Microsoft Azure Network Management services.'
-  spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
+  spec.homepage      = 'https://github.com/test-kitchen/azure-sdk-for-ruby'
```
Comment (vkarve-chef) [inline management/azure_mgmt_network/azure_mgmt_network2.gemspec]: should this be `github.com/chef/azure-sdk-for-ruby` ?

Comment (ashiqueps) [reply management/azure_mgmt_network/azure_mgmt_network2.gemspec]: Oops, sorry. Forgot to update them.

**Review by vkarve-chef — APPROVED** [review]

File: runtime/ms_rest_azure/ms_rest_azure2.gemspec [hunk 2]
```diff
@@ -7,21 +7,21 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
...
-  spec.name          = 'ms_rest_azure'
+  spec.name          = 'ms_rest_azure2'
   spec.version       = MsRestAzure::VERSION
   spec.authors       = 'Microsoft Corporation'
   spec.email         = 'azsdkteam@microsoft.com'
 
   spec.summary       = %q{Azure Client Library for Ruby.}
   spec.description   = %q{Azure Client Library for Ruby.}
-  spec.homepage      = 'https://aka.ms/ms_rest_azure'
+  spec.homepage      = 'https://github.com/chef/azure-sdk-for-ruby'
```
Comment (vkarve-chef) [inline runtime/ms_rest_azure/ms_rest_azure2.gemspec]: this one could be updated to `https://github.com/chef/azure-sdk-for-ruby/tree/master/runtime/ms_rest_azure`

Comment (ashiqueps) [reply runtime/ms_rest_azure/ms_rest_azure2.gemspec]: Done

**Pull request merged**